frameworks/g-cloud-9/questions/services/serviceCategoriesHosting.yml

```diff
@@ -1,15 +1,34 @@
+depends:
+- being: [cloud-hosting]
+  'on': lot
+hint: You can choose multiple categories.
 id: serviceCategories
 name: Service categories
-question: Which categories does your service fit under?
-hint: You can choose multiple categories.
-
-depends:
-  - "on": lot
-    being:
-      - cloud-hosting
-
-type: checkboxes
+number_of_items: 20
 options:
-  - label: Hosting type 1
-  - label: Hosting type 2
-  - label: Or maybe hosting doesn’t have categories at all?
+- {label: 'Archiving, backup and disaster recovery'}
+- {label: Compute and application hosting}
+- {label: Container service}
+- {label: Content delivery network}
+- {label: Data warehousing}
+- {label: Database - noSQL}
+- {label: Database - relational}
+- {label: Database - other}
+- {label: Distributed denial of service attack (DDOS) protection}
+- {label: Firewall}
+- {label: Infrastructure and platform security}
+- {label: Intrusion detection}
+- {label: Load balancing}
+- {label: Logging and analysis}
+- {label: Message queuing and processing}
+- {label: Networking (including network as a service)}
+- {label: Platform as a service (PaaS)}
+- {label: Protective monitoring}
+- {label: Search}
+- {label: Storage - disk (block)}
+- {label: Storage - object store}
+- {label: Storage - other}
+question: Which categories does your service fit under?
+type: checkbox_tree
+validations:
+- {message: No more than 20 categories can be selected, name: max_items_limit}
```

Comment:
Maybe we could do another validation message for empty values?
As it stands, if you answer nothing at all, you'll get back a message saying "There was a problem with the answer to this question" and I think it's probably easy for us to get a better message for this.